packages/runner/src/history.rs

```diff
@@ -62,11 +64,11 @@ mod tests {
     fn simple_work() {
         let history = DataWorkerHistory::default();
 
-        assert_eq!(history.already_received_broadcast(Some(1), 1, 1), false);
-        assert_eq!(history.already_received_broadcast(Some(1), 1, 1), true);
+        assert!(!history.already_received_broadcast(Some(1), 1, 1));
```

Comment:
same with above assert or assert_eq. More comment is I feel easier to catch in left to right order like: we need some value is true or false.